code/modules/mob/hear_say.dm

```diff
@@ -243,3 +250,9 @@
 		heard = "<span class = 'game_say'>...<i>You almost hear someone talking</i>...</span>"
 
 	src << heard
+
+/mob/proc/hear_dying(var/message, var/mob/M)
+	if(get_dist(src, M) <= 2)
+		message = "span class = 'game_say'>...You hear someone say...\"[message]\"</span>"
+	else
+		message = "<span class = 'game_say'>...<i>You almost hear someone talking</i>...</span>"
```

Comment:
You're not doing anything with these variables - they won't be sent to a user without something like `src << message`